[2022-12-04T16:12:00.111+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: data_generation.generate_data scheduled__2022-12-03T00:00:00+00:00 [queued]>
[2022-12-04T16:12:00.118+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: data_generation.generate_data scheduled__2022-12-03T00:00:00+00:00 [queued]>
[2022-12-04T16:12:00.119+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-04T16:12:00.121+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2022-12-04T16:12:00.122+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-04T16:12:00.131+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): generate_data> on 2022-12-03 00:00:00+00:00
[2022-12-04T16:12:00.138+0000] {standard_task_runner.py:55} INFO - Started process 160 to run task
[2022-12-04T16:12:00.142+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'data_generation', 'generate_data', 'scheduled__2022-12-03T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/generate_data.py', '--cfg-path', '/tmp/tmppu5oocan']
[2022-12-04T16:12:00.145+0000] {standard_task_runner.py:83} INFO - Job 3: Subtask generate_data
[2022-12-04T16:12:00.211+0000] {task_command.py:389} INFO - Running <TaskInstance: data_generation.generate_data scheduled__2022-12-03T00:00:00+00:00 [running]> on host c7dedbf79ea9
[2022-12-04T16:12:00.277+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_generation
AIRFLOW_CTX_TASK_ID=generate_data
AIRFLOW_CTX_EXECUTION_DATE=2022-12-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-03T00:00:00+00:00
[2022-12-04T16:12:00.354+0000] {python.py:177} INFO - Done. Returned value was: None
[2022-12-04T16:12:00.372+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=data_generation, task_id=generate_data, execution_date=20221203T000000, start_date=20221204T161200, end_date=20221204T161200
[2022-12-04T16:12:00.434+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-12-04T16:12:00.454+0000] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
